src/index.html

```diff
@@ -2,20 +2,69 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27'' Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <p
+        class="card__product-name"
+      >
```

Comment:
When there is only one attribute, it is not necessary to wrap it on a new line. You can simply include it on the same line as the HTML tag.
```suggestion
      <p class="card__product-name">
```